City-of-Helsinki/kukkuu-admin
# KK-629 | Add breadcrumbs to event and event group

## Description

Adds the final missing polishes for the breadcrumb feature.

## Context

[KK-629](https://helsinkisolutionoffice.atlassian.net/browse/KK-629)

## How Has This Been Tested?

I've added a browser test that checks that breadcrumbs behave correctly in the event group and event details view. These should cover the different behaviours `Breadcrumbs` can have.

## Manual Testing Instructions for Reviewers

As a logged in admin user

1. Go to event list
1. Select an event group
1. Select an event with in
1. Expect to see a breadcrumb that includes a link to the event group
1. Go to event list
1. Select an event without a event group
1. Expect to see a breadcrumb that only has a link to the event list
